<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayerList</class>
 <widget class="QDockWidget" name="LayerList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>레이어 리스트</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="margin">
     <number>3</number>
    </property>
    <item>
     <widget class="QScrollArea" name="ui_layerList_scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <widget class="QWidget" name="ui_ListContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>628</width>
         <height>406</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <property name="margin">
         <number>2</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="ui_ListVBox">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item>
           <spacer name="ui_verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ui_openTestScene">
      <property name="text">
       <string>testSceneList 열기</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ui_testScene1Select">
      <property name="text">
       <string>testScene1 선택</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ui_testScene2Select">
      <property name="text">
       <string>testScene2 선택</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ui_layerAdd">
      <property name="text">
       <string>레이어 추가</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ui_layerDelete">
      <property name="text">
       <string>레이어 삭제</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
